src/coreclr/debug/createdump/crashreportwriter.cpp

```diff
@@ -41,6 +41,7 @@ CrashReportWriter::WriteCrashReport(const std::string& dumpFileName)
         }
         WriteCrashReport();
         CloseWriter();
+        printf_status("Crash report successfully written\n");
```

Comment:
nitpick:  It would be nice if this message came in the same order as the "FAILED" message.  For example:

```
[createdump] Writing dump FAILED
[createdump] Target process is alive
```

```
[createdump] Target process is alive
[createdump] Dump successfully written
```

It would be better if the messages were always in the same order, for example like this (or vice-versa):

1. "process is alive/dead"
2. "Crash report successfully written/failed"

Though I don't think this affects anything.  We can parse it just fine, but having the ordering be the same both ways would be nice.

Reply:
It would be hard to print the alive/dead message before the crash report messages. Not impossible but it would be a lot of changes that are not suitable for a service release.